doc/user/content/get-started/quickstart.md

```diff
@@ -24,13 +24,13 @@ the superpowers of an operational data warehouse first-hand:
 
 * **Consistency**: results are always correct; never even transiently wrong.
 
-## Before you begin
+## Prerequisite
 
-All you need is a Materialize account. If you already have one —
-great! If not, [sign up for a playground account](https://materialize.com/register/?utm_campaign=General&utm_source=documentation) first.
+A Materialize account to sign into the [Materialize console](https://console.materialize.com/). If you do not have an account, you can [sign up for a free trial account](https://materialize.com/register/?utm_campaign=General&utm_source=documentation).
 
-When you're ready, head over to the [Materialize console](https://console.materialize.com/),
-and pop open the SQL Shell.
+## Step 0. Sign into the Console
```

Comment:
Isn't this a real step, anyways? You either need to check whether you're signed in, or sign in.

Reply:
Ah, yes ... but heh heh ... I made it a Step 0 so that the remaining steps will align with the quickstart instructions inside the Console.  

I'm also going back and forth whether to include it as part of the prereq, but figured when I do a pass on the quickstart content, I'll give it a better think.

Reply:
Gotcha.